includes/admin/class-amp-editor-blocks.php

```diff
@@ -11,6 +11,13 @@
  */
 class AMP_Editor_Blocks {
 
+	/**
+	 * The image size for the Latest Stories block.
+	 *
+	 * @var string
+	 */
+	const LATEST_STORIES_IMAGE_SIZE = 'large';
```

Comment:
I think this could be too big. Wouldn't `medium` be more appropriate?

Reply:
Sure, medium probably looks better, with 94b83fcd120362f75bf6d8116a95968c1bb1733a:
![latest-posts-medium](https://user-images.githubusercontent.com/4063887/53739624-d188ef80-3e57-11e9-9ceb-c357cb692997.gif)